CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 2.0.1
```

Comment:
@jakelandis There does not appear to be a gemspec for this example plugin. ???

Reply:
@dedemorton @jakelandis Interested to hear explanation for missing gemspec.

Reply:
Probably there's no gemspec because this doesn't actually get published. From the perspective of showing an example, tho, it should have a gemspec, right?

Reply:
+1 on adding gemspec. An example should be as close to real world as we can make it. 
@jakelandis  Do you agree?